### ISSUE TITLE
Title missing

### ISSUE THREAD

**Issue by AnkeSR** [klokantech/bmarchives-tickets#35]
http://www.bmarchives.org/items/show/34812

Title is in docustore but missing on the website

**Comment by andreasnef** [comment]
Hi Anke

Just checked, the title in docustore is "Vier Jahre in Asante : Tagebücher der Missionare Ramseyer und Kühne aus der Zeit ihrer Gefangenschaft" which is displayed correctly on the website. Am I looking at the wrong issue?

Best, Andreas

Am 06.03.2014 um 12:04 schrieb AnkeSR:

**Comment by AnkeSR** [comment]
Dear Andi
As you can see, D.002a, as can be seen in the link I sent, shows something totally different. So, you are and aren't looking at the wrong issue. The wrong person (Hermann Gundert - India missionary) features and there is not title, or am I seeing something different?
Let me know.

Anke

**Comment by andreasnef** [comment]
Hi Anke

I'm sorry, I don't see the issue. The given link – from my point of view – is showing the correct entry. Only the "Verlag des Misionskomptoirs" is not shown on the website, but that could be a decision taken on purpose. Maybe we can do a teamviewer session next week so I get the issue that you're trying to explain.

Enjoy the weekend,

Andreas

Am 12.03.2014 um 14:34 schrieb AnkeSR:

**Comment by andreasnef** [comment]
Indeed, it seems that the display of the title as well as the "Alternate Title" is not working in IE (tested both on Anke's computer with IE 9 as well as on another machine with IE 11). No problem on OSX with Safari or Firefox.

Andreas

Am 14.03.2014 um 10:03 schrieb Andreas Nef:

**Comment by klokan** [comment]
The issue is related to special characters in the title (and the way how they are encoded in OAI) in combination with the displaying mode of IE9+ and used OpenSans font.
We were originally testing the design in IE8, and the web runs fine there. Problems in IE9+ are new.

Technical details:

It seems the solution of the problem is to use the canonical normalized form in UTF-8 for the texts on the server side.

We are now running again the OAI-PMH harvesting of all the Docustore records (it will take a few hours from now). The records are then being reencoded into UTF-8 NKFC unicode form on our side which solved the problem during our internal testing.

Even after normalization of the characters, there are some records which causes problems in IE9+ (the text does not appear). For example word "O̲boo̲so" in: http://www.bmarchives.org/items/show/34810
with data available from:
http://46.4.125.220/oaipmh/oai2.php?verb=GetRecord&identifier=oai:m21.docuteam.ch:34810&metadataPrefix=oai_dc

This is the status of the solution after almost 3 hours of testing in different IE browsers with various forms of unicode. The mentioned records such as http://www.bmarchives.org/items/show/34812 were fixed. After the OAI reindexing is finished the whole website should be fixed.

It is still not clear how to display in IE very special unicode words such as the mentioned "O̲boo̲so".

**Comment by klokan** [comment]
Details to the unicode problems: http://msdn.microsoft.com/en-us/library/windows/desktop/dd374126(v=vs.85).aspx
